content/applications/hr/frontdesk.rst

```diff
@@ -0,0 +1,318 @@
+:guilabel:`Name` and the :guilabel:`Station` they are expected at.
+
+.. important::
+   If a guest is planned in advance, they must be checked in from the visitors' list in the
+   *Frontdesk* application. If a planned guest checks in using a kiosk, they will be checked in
+   separately, and their planned visitor entry will remain listed as :guilabel:`Planned`.
+
+   Only when a planned guest is checked-in inside the application's visitor list, their
+   :guilabel:`Planned` status changes to :guilabel:`Checked-In`.
+
+   If a guest does check-in using a kiosk, ensure all records are current, and the list of
+   currently on-site guests is correct. Be sure to check in and/or check-out the correct entries so
+   the visitor lists correctly reflects who is currently on the premises.
+
+   Ensure that planned guests are informed that they should not check-in using the kiosk if they are
+   listed as a planned guest in advance.
+
+Frontdesk flow
+==============
+
+The first thing to do after configuring the various stations is to set up each kiosk for use. It is
+recommended to use a dedicated device for each frontdesk kiosk, such as a tablet.
+
+Navigate to the kiosk in one of two ways:
+
+- Navigate to the main *Frontdesk* application dashboard, and click the :guilabel:`Open Desk`
+  button on the specific station. The kiosk loads in a new browser window.
+- Navigate to :menuselection:`Frontdesk application --> Configuration --> Stations` and click on the
+  specific station. Click the :guilabel:`Copy` button at the end of the :guilabel:`Kiosk URL` line,
+  and paste the URL into a new browser tab or window.
+
+.. important::
+   It is recommended to log out of the database and close the tab after navigating to the kiosk, so
+   there is no possibility of a visitor accessing the database when checking-in.
+
+Visitor flow
+------------
+
+When a visitor arrives at a facility, they approach a frontdesk kiosk, and enter their information.
+The information requested is what was configured for that specific frontdesk station. If any
+information is required, the field displays a red asterisk (*). The visitor must enter information
+in order to check in. Once all the information is entered, the visitor taps the :guilabel:`Check In`
+button.
+
+If drinks were configured for the station, after tapping :guilabel:`Check In`, a registration
+confirmation screen loads, along with the question `Do you want something to drink?`. The visitor
+can tap either :guilabel:`Yes, please`, or :guilabel:`No, thank you`. If they select :guilabel:`Yes,
+please`, a drink selection screen appears, and the pre-configured options are listed. The visitor
+then taps the desired selection, or if they do not want anything, they can tap the
+:guilabel:`Nothing, thanks` button at the bottom of the screen. If a drink selection was made, a
+`Thank you for registering! Your drink is on the way.` message appears.
```

Comment:
Just a general question - is there any way to change these default responses to selections on the beverage screen? Or are we stuck with them, and can't change them if desired?

Reply:
There's no way to change them that I can tell- and if it is a deep in the depths of Odoo modification, I didn't want to include it. From what I can tell, this is standard and I can't change these with anything in the application.